channel/party.go

```diff
@@ -46,6 +46,15 @@ func (d *party) addPlayer(plr *Player, index int32) {
 	d.broadcast(packetPartyPlayerJoin(d.ID, d.Name[index], d))
 }
 
+func (d *party) getPlayerID(plrID int32) byte {
```

Comment:
This should be `getPlayerPartyIndex()` or something similar. It's technically returning an index not an ID and is too similar to the prexisting playerID which represents database ID